projects/control-service/projects/pipelines_control_service/src/main/resources/application.properties

```diff
@@ -119,6 +119,10 @@ datajobs.notification.owner.name=Versatile Data Kit
 
 # The gitlab repository and credentials for pulling data jobs code when building their images.
 datajobs.git.url=${GIT_URL}
+datajobs.git.cc.grc=${GIT_GRC_URL}
```

Comment:
We can just document that git.url supports both code commit and normal git URL. 

Seems unnecessary to have both. 

All new